src/connector/FirebaseConnector.js

```diff
@@ -0,0 +1,64 @@
+const Connector = require('./Connector');
+
+class FirebaseConnector extends Connector {
+    constructor(firebaseClient, rootRef) {
+        super();
```

Comment:
Can you please add a comment explaining what rootRef is?

Reply:
This is the name of the root key of the firebase database.